infra/terraform/test_fixtures/terraform-google-iam.tf

```diff
@@ -0,0 +1,80 @@
+locals {
+  // Roles required by CI service accounts to run the IAM
+  // integration tests.
+  iam_required_roles = [
+    "roles/resourcemanager.organizationAdmin",
+    "roles/compute.xpnAdmin",
+    "roles/storage.admin",
+    "roles/pubsub.admin",
+    "roles/cloudkms.admin",
+  ]
+}
+
+resource "google_project" "iam" {
+  provider = "google.phoogle"
+  name = "ci-iam"
+  project_id = "ci-iam"
+  folder_id = "${google_folder.phoogle_cloud_foundation_cicd.name}"
+  billing_account = "${module.variables.phoogle_billing_account}"
+}
+
+resource "google_project_services" "iam" {
+  provider = "google.phoogle"
+
+  project = "${google_project.iam.id}"
+  services = [
+    "admin.googleapis.com",
+    "appengine.googleapis.com",
+    "cloudbilling.googleapis.com",
+    "cloudresourcemanager.googleapis.com",
+    "compute.googleapis.com",
+    "iam.googleapis.com",
+    "iamcredentials.googleapis.com",
+    "oslogin.googleapis.com",
+    "serviceusage.googleapis.com",
+  ]
+}
+
+resource "google_service_account" "iam" {
+  provider = "google.phoogle"
+  project      = "${google_project.iam.id}"
+  account_id   = "ci-iam"
+  display_name = "ci-iam"
+}
+
+resource "google_folder_iam_member" "iam" {
+  provider = "google.phoogle"
+  count = "${length(local.iam_required_roles)}"
+  folder = "${google_folder.phoogle_cloud_foundation_cicd.name}"
+  role   = "${element(local.iam_required_roles, count.index)}"
```

Comment:
Only the roles related to folders should be applied at the folder level.

Reply:
Addressed in 4e81556 and augmented in bc4c440